fa25-team-a/analysis/Ria_Q3_PriceTier_Dataset.ipynb

```
@@ -0,0 +1,165 @@
+        "\n",
+        "This notebook performs the following tasks on the full merged CHAPA resale dataset:\n",
+        "\n",
+        "1. Drop rows with missing `Town` or `Address`.\n",
+        "2. Clean and convert `Maximum Resale Price` to numeric.\n",
+        "3. Fix encoding issues in `income_bracket` and `asset_bracket`.\n",
+        "4. Identify and remove invalid or zero resale prices.\n",
+        "5. Create 4-tier price categories based on quartiles.\n",
+        "6. Save the cleaned dataset.\n",
+        "7. Optional: Show summary statistics for each price tier.\n"
+      ],
+      "metadata": {
```

Ruff (0.14.5)

String contains ambiguous `‚` (SINGLE LOW-9 QUOTATION MARK). Did you mean `,` (COMMA)?

(RUF001)